### PR TITLE
Add `create-bounties.md`

### PR DESCRIPTION
**Pull request by Smephite**

### CODE REVIEW

File: bounties/level-1/create-bounties.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Creating Bounty Suggestions
+
+| Hunter | Agent
+| :- | :-
+| 500 XLM per bounty | 0 Credits (success will be determined by PR acceptance)
```
Comment (tyvdh) [inline bounties/level-1/create-bounties.md:5]: Let's decrease this to 200 XLM. Bounties haven't been as tricky to create as I first suspected. I'll honor the 500 XLM retrospectively but ongoing this should be lower.

**Review by tyvdh — CHANGES_REQUESTED** [review]
This PR is likely dependent on the bounty template PR being completed and accepted first as this would be necessary before more folks start suggesting bounties. So this bounty should likely link to that bounty with the idea that one needs to be completed first.

**Comment by ralphilius** [comment]
What about criteria to decide bounty prize and credits? Size of impact? Or size of effort?

**Comment by Smephite** [comment]
I'd say size of effort

**Review by tyvdh — CHANGES_REQUESTED** [review]
Would you mind making the necessary adjustments such to meet with our new standards as detailed here? https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/README.md

If there are things missing from that standard please make update requests in this issue https://github.com/tyvdh/stellar-quest-bounties/issues/40 first.

Thanks! Looking forward to getting everyone paid for all their great bounty suggestions over the past couple weeks.

File: bounties/level-1/create-bounties.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Creating Bounty Suggestions
+
+## Award
+| Hunter | Agent | Type
+| :- | :- | :-
+| 200 XLM per submission | 0 Credits | Continuous
+
+## Context
+https://github.com/tyvdh/stellar-quest-bounties/issues?q=create-bounties.md
```
Comment (tyvdh) [inline bounties/level-1/create-bounties.md:9]: Let's exclude the .md suffix just in the chance folks don't use the .md extension.

File: bounties/level-1/create-bounties.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+* The level and payout should be determined by difficulty / effort of the task given.  
+
+* Typical payouts and difficulty assessments for each level can be found in the related `level-{n}/README.md`
+  - [Level 1](https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-1/README.md)
+  - [Level 2](https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-2/README.md)
+  - [Level 3](https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-3/README.md)
+
+* The submitted Markdown file must be placed in the folder of the given difficulty and named after the bounty title. Example given: `level-{n}/my-bounty-name.md`.
+
+Overall there is a lot of freedom regarding possible quests therefor: be creative!
```
Comment (tyvdh) [inline bounties/level-1/create-bounties.md:36]: therefor -> therefore 😁 

https://www.grammarly.com/blog/therefore-vs-therefor/?&utm_source=google&utm_medium=cpc&utm_campaign=10273012991&utm_targetid=aud-630734890445:dsa-1233402314764&gclid=CjwKCAjw0qOIBhBhEiwAyvVcf4GpBFE-wJcW3LksecGGpnotF0EzlzA5NVuJSc-9u9q6bk3R6HO6iBoCu_IQAvD_BwE&gclsrc=aw.ds

File: bounties/level-1/create-bounties.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+#### Review Criteria
+  - The bounty is considered to be complete as soon as the PR is approved and merged.
+
+## Links
+
+- https://github.com/tyvdh/stellar-quest-bounties
+
+## Example
+- https://github.com/tyvdh/stellar-quest-bounties/pull/13
+- https://github.com/tyvdh/stellar-quest-bounties/pull/9
```
Comment (tyvdh) [inline bounties/level-1/create-bounties.md:51]: Just for beauty sake let's format this consistently with spacing. Line 41 butts right against 42, there should be a space. 43 has a tab, 47, 50 and 51 do not. There's a space between ##Links and the first link but that pattern isn't used elsewhere. I'm a stickler for clean consistent code.

**Review by tyvdh — CHANGES_REQUESTED** [review]
Really nice! Just a couple more items and I think we're _finally_ ready to ship this!

File: bounties/level-1/create-bounties.md (hunk 3 above)
Comment (Smephite) [reply bounties/level-1/create-bounties.md:36]: Still confused but 🤷 🤣

**Pull request merged**